### DISCUSSION TITLE
Can I turn on Class as text in tooltip?

### DISCUSSION THREAD

**Discussion by tnwn** [kaytotes/ImprovedBlizzardUI#173] [Q&A]
As the addon works now I can't seem to find any way to tell what class another player or NPC is other than by the coloring system. I never remember those so I'd rather have it in text.

Is there a way to do this that I haven't found out yet or is it not implemented in this addon?

**Comment by kaytotes** [comment]
This is not currently implemented. I'm not opposed to adding it however.

**Reply by tnwn** [reply]
Thanks for answering. Then at least I know I'm not blind when I can't find it...

It's not entirely crusial for using the UI,  but sometimes it'd be nice to see what class someone is and I never remember the color system, as I saied :)
